x-pack/legacy/plugins/reporting/server/lib/validate/validate_config.ts

```diff
@@ -5,15 +5,19 @@
  */
 
 import crypto from 'crypto';
-import { Logger } from '../../../types';
+import { ServerFacade, Logger } from '../../../types';
+
```

Comment:
This is a tough one since it's setting and defaulting a config... I wonder if we can push this in our joi schema somehow?

EDIT: This was said with the thought that we should insulate ourselves from the `server` object as much as possible.

Reply:
I agree, it is tough because we could do more here to protect the environment from bad things happening. But giving them a made-up encryptionKey on the fly does help the first-time user experience.